lib/update/engine.go

```diff
@@ -87,7 +85,13 @@ func (f *fsmUpdateEngine) GetExecutor(p fsm.ExecutorParams, remote fsm.Remote) (
 // RunCommand executes the phase specified by params on the specified server
 // using the provided runner
 func (f *fsmUpdateEngine) RunCommand(ctx context.Context, runner fsm.RemoteRunner, server storage.Server, p fsm.Params) error {
-	args := []string{"upgrade", "--phase", p.PhaseID, fmt.Sprintf("--force=%v", p.Force)}
```

Comment:
`--force=<true|false>` always turns `--force` on. To turn it off (i.e. force=false), it must be omitted.